tests/org.eclipse.ui.workbench.texteditor.tests/META-INF/MANIFEST.MF

```diff
@@ -18,9 +18,9 @@ Require-Bundle:
  org.eclipse.ui;bundle-version="[3.5.0,4.0.0)",
  org.junit;bundle-version="4.12.0",
  org.eclipse.text.tests;bundle-version="[3.5.0,4.0.0)",
- org.eclipse.core.expressions;bundle-version="[3.5.0,4.0.0)"
+ org.eclipse.core.expressions;bundle-version="[3.5.0,4.0.0)",
+ org.mockito.mockito-core
 Bundle-RequiredExecutionEnvironment: JavaSE-17
 Eclipse-BundleShape: dir
 Automatic-Module-Name: org.eclipse.ui.workbench.texteditor.tests
-Import-Package: org.mockito,
- org.mockito.stubbing;version="5.5.0"
+Import-Package: org.mockito.stubbing;version="[5.11.0,6.0.0)"
```

Comment:
It look like these dependency changes are not necessary, are they? So please revert them if possible or otherwise please stick to package imports.